EventFiltering/PWGHF/HFFilter.cxx

```diff
@@ -392,8 +403,14 @@ struct HfFilter { // Main struct for HF triggers
       hProtonTOFPID = registry.add<TH2>("fProtonTOFPID", "#it{N}_{#sigma}^{TOF} vs. #it{p} for selected protons;#it{p} (GeV/#it{c});#it{N}_{#sigma}^{TOF}", HistType::kTH2F, {{100, 0., 10.}, {200, -10., 10.}});
     }
 
+    ccdb->setURL(url.value);
+    ccdb->setCaching(true);
+    ccdb->setLocalObjectValidityChecking();
+    ccdb->setCreatedNotAfter(std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count());
+    ccdbApi.init(url);
+
     // init ONNX runtime session
-    if (applyML) {
+    if (applyML && ccdbTimestamp > 0) {
```

Comment:
> I would keep the possibility to read models from cvmfs (at least for the time being)

yes, but I would just keep "&& ccdbTimestamp != 0", otherwise there will be a error when ccdbTimestamp ==0